rfcs/0003-xcmp-picasso-statemine.md

```diff
@@ -0,0 +1,252 @@
+
+# Overview
+
+This proposal sugget to open bidirectional HRMP channel between Picasso and Statemine. This will enable crosschain communication between Picasso and Statemine to enable various use cases including crosschain token transfer.
```

Comment:
Is there any particular roadmap that describes what use cases Picasso is trying to address with this channel? Or is just about token transfers of KSM to Picasso?